<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormatPerformerTagsOptionsPage</class>
 <widget class="QWidget" name="FormatPerformerTagsOptionsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>637</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>360</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>window.title</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>731</width>
        <height>635</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="page_title">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>page.title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="page_description">
         <property name="text">
          <string>page.text</string>
         </property>
         <property name="textFormat">
          <enum>Qt::MarkdownText</enum>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>6</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="section_keyword_title">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>section.keywords.title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="section_keyword_frame">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QGroupBox" name="group_additonal">
            <property name="title">
             <string>section.keyword.additional.title</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <item>
              <widget class="QRadioButton" name="additional_rb_1">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="additional_rb_2">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string notr="true">2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="additional_rb_3">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string notr="true">3</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="additional_rb_4">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string notr="true">4</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="group_guest">
            <property name="title">
             <string>section.keyword.guest.title</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <item>
              <widget class="QRadioButton" name="guest_rb_1">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="guest_rb_2">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string notr="true">2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="guest_rb_3">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string notr="true">3</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="guest_rb_4">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string notr="true">4</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="group_solo">
            <property name="title">
             <string>section.keyword.solo.title</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <item>
              <widget class="QRadioButton" name="solo_rb_1">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="solo_rb_2">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string notr="true">2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="solo_rb_3">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string notr="true">3</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="solo_rb_4">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string notr="true">4</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="group_vocals">
            <property name="title">
             <string>section.keyword.vocals.title</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="leftMargin">
              <number>1</number>
             </property>
             <property name="topMargin">
              <number>1</number>
             </property>
             <property name="rightMargin">
              <number>1</number>
             </property>
             <property name="bottomMargin">
              <number>1</number>
             </property>
             <item>
              <widget class="QRadioButton" name="vocals_rb_1">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="vocals_rb_2">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string notr="true">2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="vocals_rb_3">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string notr="true">3</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="vocals_rb_4">
               <property name="maximumSize">
                <size>
                 <width>40</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string notr="true">4</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>6</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="section_display_title">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>section.display.title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="section_display_frame">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <property name="horizontalSpacing">
             <number>10</number>
            </property>
            <property name="verticalSpacing">
             <number>1</number>
            </property>
            <item row="3" column="1" alignment="Qt::AlignHCenter">
             <widget class="QLineEdit" name="format_group_3_start_char">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string notr="true"> (</string>
              </property>
              <property name="placeholderText">
               <string>placeholder.blank</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2" alignment="Qt::AlignHCenter">
             <widget class="QLineEdit" name="format_group_3_sep_char">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="placeholderText">
               <string>placeholder.blank</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" alignment="Qt::AlignLeft">
             <widget class="QLabel" name="label_section_1">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>section.display.label.section_1</string>
              </property>
             </widget>
            </item>
            <item row="3" column="3" alignment="Qt::AlignHCenter">
             <widget class="QLineEdit" name="format_group_3_end_char">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string notr="true">)</string>
              </property>
              <property name="placeholderText">
               <string>placeholder.blank</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" alignment="Qt::AlignLeft">
             <widget class="QLabel" name="label_section_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>section.display.label.section_2</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1" alignment="Qt::AlignHCenter">
             <widget class="QLineEdit" name="format_group_4_start_char">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string notr="true"> (</string>
              </property>
              <property name="placeholderText">
               <string>placeholder.blank</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2" alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_sep">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>section.display.label.sep</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" alignment="Qt::AlignHCenter">
             <widget class="QLineEdit" name="format_group_2_start_char">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string notr="true">, </string>
              </property>
              <property name="placeholderText">
               <string>placeholder.blank</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2" alignment="Qt::AlignHCenter">
             <widget class="QLineEdit" name="format_group_1_sep_char">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="placeholderText">
               <string>placeholder.blank</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" alignment="Qt::AlignHCenter">
             <widget class="QLineEdit" name="format_group_1_start_char">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="placeholderText">
               <string>placeholder.blank</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" alignment="Qt::AlignLeft">
             <widget class="QLabel" name="label_section_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>section.display.label.section_4</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3" alignment="Qt::AlignHCenter">
             <widget class="QLineEdit" name="format_group_2_end_char">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="placeholderText">
               <string>placeholder.blank</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" alignment="Qt::AlignLeft">
             <widget class="QLabel" name="label_section_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>section.display.label.section_3</string>
              </property>
             </widget>
            </item>
            <item row="4" column="3" alignment="Qt::AlignHCenter">
             <widget class="QLineEdit" name="format_group_4_end_char">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string notr="true">)</string>
              </property>
              <property name="placeholderText">
               <string>placeholder.blank</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2" alignment="Qt::AlignHCenter">
             <widget class="QLineEdit" name="format_group_2_sep_char">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="placeholderText">
               <string>placeholder.blank</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3" alignment="Qt::AlignHCenter">
             <widget class="QLineEdit" name="format_group_1_end_char">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="placeholderText">
               <string>placeholder.blank</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_start">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>section.display.label.start</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3" alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_end">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>section.display.label.end</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2" alignment="Qt::AlignHCenter">
             <widget class="QLineEdit" name="format_group_4_sep_char">
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="placeholderText">
               <string>placeholder.blank</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>6</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="section_example_title">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>section.examples.title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="section_examples_frame">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="example_instruments">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="example_vocals">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
